frontend/test/parameters/components/widgets/CategoryWidget.integ.spec.js

```diff
@@ -0,0 +1,79 @@
+import "__support__/integrated_tests";
+
+import React from "react";
+
+import CategoryWidget from "metabase/parameters/components/widgets/CategoryWidget";
+
+import { mount } from "enzyme";
+import {
+    click, clickButton
+} from "__support__/enzyme_utils"
+
+const VALUES = [
+    ["First"],
+    ["Second"],
+    ["Third"],
+];
+
+const ON_SET_VALUE = jest.fn();
+
+function renderCategoryWidget(props) {
+    return mount(<CategoryWidget
+                    values={VALUES}
+                    setValue={ON_SET_VALUE}
+                    onClose={() => {}}
+                    {...props}
+                />
+    );
+}
+
+describe("CategoryWidget", () => {
```

Comment:
Nicely written tests! 😊

This could include a test about this bug I wrote to earlier comment: "If you have already applied a filter, removing the filter by unchecking the applied values and clicking "Done" leaves the filter in a weird state"